src/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export module InteractiveConstants {
```

Comment:
Where else could you place such information? 🤔 As a static method in Scenes?
Also that way other constants for stuff could be grouped.